### ISSUE TITLE
Display custom nicknames in Draw Full Names mode

### ISSUE THREAD

**Issue by MaximeCouty** [DanielAbalde/Sunglasses#16]
Hi Daniel,

Small glitch that I noticed when playing with this plugin. 
In Draw Full Names mode (which I use), the display custom nicknames only works for components that have identical Names and Nicknames.

Example below with Partition List whose nickname is _Partition_, but displayed as _Partition List_ when GH is in draw full names mode.

![image](https://user-images.githubusercontent.com/72132924/219042936-d2417a10-abf7-49a1-8719-4a327f977540.png)

I had a look at the code and I'm not sure if this is sufficient, but maybe the nickname should be also compared to the name of the component to determine is the filtering has to occur or not.

**Comment by DanielAbalde** [comment]
Hi there, thanks for the notice.

`Partition` is the default nickname of `Partition List`, so it's not a "custom" or user-defined nickname, that's why it's not displayed when `Display only custom nicknames` is active. You can solve this by including a whitespace at the end or start, for example `Partition `.

**Comment by MaximeCouty** [comment]
Yes, I noticed that, but actually what I would like is that `Partition List` does NOT show on top of the component.
From what I understand the Draw Full Names mode overrides the nickname and puts the name instead, though the default Nickname is unchanged. What I'm suggesting is that Sunglasses should compare to the name and the nickname, and if either is a match, then the text display is disabled.
